server/scripts/start_blockchain.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
+# This script is creating a new chain and setting up the services needed to run
+# an evoting system. It ends by starting the http server needed by the frontend
```

Comment:
evoting system is probably not what we want, right? 😜

Reply:
whoopsy